src/GraphQL/DataObjectType/PimcoreObjectType.php

```diff
@@ -87,6 +87,10 @@ public function build($context = [])
             [
                 'type' => Type::id(),
             ],
+            'fullpath' => [
```

Comment:
I don't think that we should expose this by default. You can still use the configurator to add it.

Reply:
I removed it again.